lib/util/device-service-details.ts

```diff
@@ -19,6 +19,7 @@ export interface CurrentService {
 
 export interface CurrentServiceWithCommit extends CurrentService {
 	commit: string;
+	commitId: number;
```

Comment:
```suggestion
	release_id: number;
```
So that it's clear what id this is for.